### PR TITLE
feat: convert backend netcore docs

### PR DESCRIPTION
**Pull request by MarkComerford**
#### 📲 What

Converted backend netcore docs to asciidoc

#### 🤔 Why
		
Why it's needed, background context.
		
#### 🛠 How
		
More in-depth discussion of the change or implementation.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR.
		 
#### 🕵️ How to test

Notes on how a reviewer can test the changes, e.g. how to run the tests.

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: asciidoc/workloads/azure/backend/netcore/introduction_netcore.adoc [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            ]
+        }
+    ],
+    "enabled": true
+}
+----
+
+Even though the restaurant/menu context is used as the foundation for the templates, the .NET template engine provides us with the ability to change the domain with a simple command. The templates are highly customizable and modular, so a project can be generated with only what you need. Conditional compilation is used depending on the settings you provide to generate the minimal amount of code needed for your project.
+
+image::../../../src/components/HideNavigation/HideNavigation.png[Hide Navigation,align=right]
```
Comment (leesaxby) [inline asciidoc/workloads/azure/backend/netcore/introduction_netcore.adoc:51]: This image doesn't seem to be rendering?

Comment (leesaxby) [reply asciidoc/workloads/azure/backend/netcore/introduction_netcore.adoc:51]: Could try declaring it at the top of the file.
`:imagesdir: ../../../src/components/HideNavigation`

Comment (leesaxby) [reply asciidoc/workloads/azure/backend/netcore/introduction_netcore.adoc:51]: Actually do we need this image, is this just hiding the docusaurus nav buttons?

Comment (MarkComerford) [reply asciidoc/workloads/azure/backend/netcore/introduction_netcore.adoc]: Checked the current page. There isn't any image, so just removed this.

**Review by leesaxby — APPROVED** [review]

**Pull request merged**